src/app/admin/tests/[testsuite_id]/page.tsx

```diff
@@ -389,6 +390,51 @@ export default function Page({ params }: { params: { testsuite_id: string } }) {
     });
   };
 
+  const papaparseOptions = {
```

Comment:
Let's call it `csvParseOptions`, it's more intuitive that way, even though I agree it's options for `papaparse` library, this variable name might not communicate what's the purpose of papaparse.

Reply:
ok